cmd/ipfs-cluster-ctl/main.go

```diff
@@ -277,6 +277,10 @@ If you prefer faster adding, add directly to the local IPFS and trigger a
 					Name:  "quieter, Q",
 					Usage: "Write only final hash to output",
 				},
+				cli.BoolFlag{
+					Name:  "no-stream",
+					Usage: "Buffer output locally. Produces a valid slice with --enc=json.",
```

Comment:
hmm good catch. I changed the name of the flag and introduced a bug. The `--no-stream` output should produce a `[ .. ]` result printed at the end. The streaming output prints AddedOuput object as they arrive.

I want to name it `no-stream` because it relates to how results are streamed to the user. And might eventually mean that this will use no-streaming endpoint from the server in the future.

Reply:
> why not always buffer locally?

In the general case, you are just using memory on stuff that you can flush out. Mid-term plan will be to move any endpoints listing things to streaming, as otherwise it's unpractical to deal with huge pinsets.